api/types.go

```diff
@@ -154,17 +158,21 @@ type GenericTicketing struct {
 }
 
 type JiraClient struct {
-	Url      string         `json:"url,omitempty"`
-	Username kommons.EnvVar `yaml:"username" json:"username"`
-	Password kommons.EnvVar `yaml:"password" json:"password"`
+	Url      string            `json:"url,omitempty"`
+	Username kommons.EnvVar    `yaml:"username" json:"username"`
+	Password kommons.EnvVar    `yaml:"password" json:"password"`
+	Values   ResponderValues   `json:"values"`
```

Comment:
I think we can put this into a common struct, and then embed with `json:",inline"` - also no need to create a custom type for the map[string]string